okteto/okteto
# Split integration tests into jobs

# Proposed changes

Fixes https://okteto.atlassian.net/browse/DEV-507

- remove integration commands, instead run jobs for each of them, this way we can retrigger a job without having to retrigger the whole workflow
- when labels on PRs are used to run integration, the new workflows are triggered without duplicating the `lint-test-build` launched by the push to the branch
- use okteto context instead of deprecated okteto login on integration jobs and okteto analytics --disable instead of creating the file
- reduce binary build time by using exclusive cache key, when building the windows binary the pkg where not saved at the cache key used on that job (reduced 5 minuts)


How to test:
- create new branch from this and make some code changes, push and check workflows
- trigger with `run-e2e` labels (unix and windows) the integration workflows are triggered
- create a branch that ends with win or e2e to check workflows are triggered for integration
